addons/binding/org.openhab.binding.chromecast/ESH-INF/thing/thing-types.xml

```diff
@@ -4,6 +4,34 @@
         xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
         xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
 
+    <!-- Chromecast Audio Group Thing Type -->
+    <thing-type id="audiogroup">
+        <label>Chromecast Audio Group</label>
+        <description>A Google Chromecast Audio Group device</description>
+
+        <channels>
+            <channel id="control" typeId="control"/>
+            <channel id="volume" typeId="volume"/>
+            <channel id="mute" typeId="mute"/>
+            <channel id="playuri" typeId="playuri"/>
+        </channels>
+
+        <config-description>
+            <parameter name="ipAddress" type="text">
+                <context>network-address</context>
+                <label>Network Address</label>
+                <description>Network address of the Chromecast device.</description>
+                <required>true</required>
+            </parameter>
+            <parameter name="port" type="integer">
+                <context>network-port</context>
+                <label>Network Port</label>
+                <description>Network port of the Chromecast device.</description>
+                <required>true</required>
```

Comment:
This would almost always be discovered but added a default if created manually (same a milight binding where it is required but a default is provided).